lib/reek/cli/options.rb

```diff
@@ -4,6 +4,7 @@
 require 'rainbow'
 require_relative '../version'
 require_relative 'status'
+require_relative '../../../lib/reek/detector_repository'
```

Comment:
You can leave out `../../lib/reek/`.

Reply:
Whoops, that's what I get for using Rubymine's autocompletion :)